src/encoded/tests/test_reports_metadata.py

```diff
@@ -1155,10 +1259,9 @@ def test_metadata_metadata_report_build_new_request(dummy_request):
         '&field=files.paired_with&field=files.index_of&field=files.derived_from&field=files.file_size'
         '&field=files.lab.title&field=files.md5sum&field=files.dbxrefs&field=files.href'
         '&field=files.genome_annotation&field=files.platform.title&field=files.controlled_by'
-        '&field=files.status&field=files.no_file_available&field=files.restricted'
         '&field=files.s3_uri&%40id=%2Fexperiments%2FENCSR123ABC%2F'
     )
-    new_request.effective_principals == ['system.Everyone']
+    assert new_request.effective_principals == ['system.Everyone']
```

Comment:
I don't understand how this is a success?  Just a ways to see if the new Request instantiated?

Reply:
`effective_principals` is a list that should equal `['system.Everyone']`. Was missing its assert statement.